modules/subscriptions.php

```diff
@@ -667,25 +667,34 @@ function widget( $args, $instance ) {
 			</form>
 
 			<script>
-				( function( d ) {
-					if ( ( 'placeholder' in d.createElement( 'input' ) ) ) {
-						var label = d.getElementById( 'jetpack-subscribe-label' );
-	 					label.style.clip 	 = 'rect(1px, 1px, 1px, 1px)';
-	 					label.style.position = 'absolute';
-	 					label.style.height   = '1px';
-	 					label.style.width    = '1px';
-	 					label.style.overflow = 'hidden';
-					}
-				} ) ( document );
-
-				// Special check for required email input because Safari doesn't support HTML5 "required"
-				jQuery( '#subscribe-blog-<?php echo $widget_id; ?>' ).submit( function( event ) {
-					var requiredInput = jQuery( this ).find( '.required' );
-					if ( requiredInput.val() == '' ) {
-						event.preventDefault();
-						requiredInput.focus();
-					}
-				});
+			( function( d ) {
+				if ( ( 'placeholder' in d.createElement( 'input' ) ) ) {
+					var label = d.getElementById( 'jetpack-subscribe-label' );
+						label.style.clip 	 = 'rect(1px, 1px, 1px, 1px)';
+						label.style.position = 'absolute';
+						label.style.height   = '1px';
+						label.style.width    = '1px';
+						label.style.overflow = 'hidden';
+				}
+			
+				var form = d.getElementById('subscribe-blog-<?php echo $widget_id; ?>'), 
+					input = d.getElementById('<?php echo esc_attr( $subscribe_field_id ); ?>'),
+					handler = function(e) {
+						if (input.value === '') {
+							input.focus();
+							
+							if (e.preventDefault)
```

Comment:
Thanks for fixing the issue, could you go a little bit further and make the code you have written adhere to the coding standards? For example, you have no spaces inside braces and the if on this line should have curly braces even though it's a one-liner. Here are the standards we use: https://make.wordpress.org/core/handbook/coding-standards/javascript/